op-service/sources/batching/call.go

```diff
@@ -117,3 +117,7 @@ func (c *CallResult) GetAddress(i int) common.Address {
 func (c *CallResult) GetBigInt(i int) *big.Int {
 	return *abi.ConvertType(c.out[i], new(*big.Int)).(**big.Int)
 }
+
+func (c *CallResult) GetStruct(i int, target interface{}) {
+	abi.ConvertType(c.out[i], target)
```

Comment:
I'm concerned that if the ABI changes, this will silently succeed and it'll be hard to detect. Is that the case here?

Reply:
No, if the contract changes the tests will fail. Sadly they'll panic which is a little excessive but you definitely can't miss it. Same is true for the other `Get*` functions - if you try to use `GetBigInt` to get a `uint8` it will panic. The generated bindings will give the same panic if the contract and bindings are out of sync as they use the same `ConvertType` function.

Reply:
Also worth noting that if the struct gets out of sync, we'll get failures from `faultdisputegame_test` because it will fail to pack the output the test provides which provides a much nicer failure message than just having a test panic unexpectedly.

Reply:
I wasn't sure that it won't even panic. A panic is fine here.